src/hpack/encoder.rs

```diff
@@ -443,6 +446,23 @@ mod test {
         assert_eq!(1, res.len());
     }
 
+    #[test]
+    fn test_encode_indexed_name_literal_value() {
+        let mut encoder = Encoder::default();
+        let res = encode(&mut encoder, vec![header("content-language", "foo")]);
+
+        assert_eq!(res[0], 0b01000000 | 27); // Indexed name
+        assert_eq!(res[1], 0x80 | 2); // header value w/ huffman coding
+
+        assert_eq!("foo", huff_decode(&res[2..4]));
+
+        // Same name, new value should still use incremental
+        let res = encode(&mut encoder, vec![header("content-language", "bar")]);
+        assert_eq!(res[0], 0b01000000 | 27); // Indexed name
```

Comment:
The seed I was using to replay the fuzz had this indexed name, and I noticed that new values were pointing at the dynamic index when it could be pointing at the static index. The HPACK spec has [an example](https://httpwg.org/specs/rfc7541.html#n-third-response_1) doing exactly this (well, the `date` header...).